{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "DEBUG", "message": "debug message", "logger": "__main__", "module": "main", "function": "main", "line": 15, "thread_name": "MainThread", "acdc": "sample line"}
{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "INFO", "message": "info message", "logger": "__main__", "module": "main", "function": "main", "line": 16, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "WARNING", "message": "warning message", "logger": "__main__", "module": "main", "function": "main", "line": 17, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "ERROR", "message": "error message", "logger": "__main__", "module": "main", "function": "main", "line": 18, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "CRITICAL", "message": "critical message", "logger": "__main__", "module": "main", "function": "main", "line": 19, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:08:27.855372+00:00", "level": "ERROR", "message": "exception message:\nTraceback (most recent call last):\n  File \"D:\\!wip\\fellerbuncher\\main.py\", line 22, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero", "logger": "__main__", "module": "main", "function": "main", "line": 24, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "DEBUG", "message": "debug message", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 15, "thread_name": "MainThread", "acdc": "sample line"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "INFO", "message": "info message", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 16, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "WARNING", "message": "warning message", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 17, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "ERROR", "message": "error message", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 18, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "CRITICAL", "message": "critical message", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 19, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:43.265879+00:00", "level": "ERROR", "message": "exception message:\nTraceback (most recent call last):\n  File \"D:\\!wip\\fellerbuncher\\main_fellerbuncher.py\", line 22, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero", "logger": "__main__", "module": "main_fellerbuncher", "function": "main", "line": 24, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "DEBUG", "message": "debug message", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 24, "thread_name": "MainThread", "acdc": "sample line"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "INFO", "message": "info message", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 25, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "WARNING", "message": "warning message", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 26, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "ERROR", "message": "error message", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 27, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "CRITICAL", "message": "critical message", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 28, "thread_name": "MainThread"}
{"timestamp": "2025-04-21T02:09:51.653625+00:00", "level": "ERROR", "message": "exception message:\nTraceback (most recent call last):\n  File \"D:\\!wip\\fellerbuncher\\main_setup_logging.py\", line 31, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero", "logger": "__main__", "module": "main_setup_logging", "function": "main", "line": 33, "thread_name": "MainThread"}
